### ISSUE TITLE
run supports "shell" option

### ISSUE THREAD

**Issue by shaddollxz** [huochunpeng/makes#19]
wrong: while use a script whth "\*" in after task ,got an error ,it is becase `options.shell` of `cross-spawn`  is `false` by default

expect:  `run` support a "shell" option like `cross-spawn`

```javascript
export default async function ({ run }) {
    await run("chmod", ["+x", ".husky/*"]); // chmod: .husky/*: No such file or directory
}
```

**Comment by 3cp** [comment]
Thank you!

**Comment by 3cp** [comment]
Shipped in v3.1.5.
